pkgs/development/python-modules/clifford/default.nix

```
@@ -21,6 +22,10 @@ buildPythonPackage rec {
     inherit pname version;
     sha256 = "ade11b20d0631dfc9c2f18ce0149f1e61e4baf114108b27cfd68e5c1619ecc0c";
   };
+  patches = [ (fetchpatch {
+    url = "https://github.com/pygae/clifford/pull/388.patch";
+    sha256 = "0pkpwnk0kfdxsbzsxqlqh8kgif17l5has0mg31g3kyp8lncj89b1";
+  }) ];
```

do you mind adding a comment and a suggestion in which it can be removed.
```suggestion
  patches = [
    # fix xyz, can be removed  next bump
    (fetchpatch {
      url = "https://github.com/pygae/clifford/pull/388.patch";
     sha256 = "0pkpwnk0kfdxsbzsxqlqh8kgif17l5has0mg31g3kyp8lncj89b1";
    })
  ];
```